extensions-core/s3-extensions/src/main/java/io/druid/storage/s3/S3DataSegmentPusherConfig.java

```diff
@@ -25,6 +25,8 @@
  */
 public class S3DataSegmentPusherConfig
 {
+  public static final int MAX_LISTING_LENGTH = 1000;
```

Comment:
I went through other configuration classes and found out `@Min` would be good to use here. I've done with it.